test/cmd/rbac.sh

```diff
@@ -30,6 +30,12 @@ run_clusterroles_tests() {
   kube::test::get_object_assert clusterrolebindings/cluster-admin "{{.metadata.name}}" 'cluster-admin'
 
   # test `kubectl create clusterrole`
+  kubectl create "${kube_flags[@]}" clusterrole pod-admin --verb=* --resource=pods
+  kube::test::get_object_assert clusterrole/pod-admin "{{range.rules}}{{range.verbs}}{{.}}:{{end}}{{end}}" '\*:'
+  output_message=$(! kubectl delete clusterrole pod-admin -n test 2>&1 "${kube_flags[@]}")
```

Comment:
the command shouldn't fail, so there shouldn't be a `!` in this invocation, right?

Reply:
Correct